modules/installing-using-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,158 @@
+====
+
+Follow the steps in the procedure to understand how to use the Assisted Installer to install a {product-title} cluster on OCI. The procedure demonstrates the use of the {oci}'s Manager (CCM) and Oracle’s Container Storage Interface (CSI) objects to link your {product-title} cluster with {oci}'s API.
+
+.Prerequisites
+
+* You reviewed details about the {product-title} xref:../../architecture/architecture-installation.adoc#architecture-installation[installation and update] processes. 
+//* You completed the [Request Access to Red Hat OpenShift on Oracle Cloud Infrastructure in Developer Preview](https://docs.google.com/forms/d/e/1FAIpQLSdQHmw0BUWjQxPKC5_G7lk2_o9Pcnn2ON84Al6s2Ir254gv6Q/viewform) form.
+* If you use a firewall and you plan to use a Telemetry service, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured your firewall], so that {product-title} can access the sites required.
+* You configured an {oci} account to host the cluster. See link:https://docs.oracle.com/en/cloud/get-started/subscriptions-cloud/csgsg/create-users-and-assign-roles.html#GUID-1041AC78-9E43-42CF-B5A5-73F8B8FEB39A[Create Users and Assign Roles] in the Oracle documentation. 
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can manually create and maintain IAM credentials. See link:https://docs.oracle.com/en-us/iaas/Content/Identity/Concepts/overview.htm[Overview of Identity and Access Management] in the Oracle documentation. 
+* You set up your cloud infrastructure, such as compute, networking, and storage settings, before installing {product-title}. You must ensure that your infrastructure meets the minimum Assisted Installer link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2024/html-single/assisted_installer_for_openshift_container_platform/index#hardware[hardware requirements]. 
+
+Before you create your VMs, refer to {op-system-base-full} certified shapes, instance types, to identify the supported {oci} VM shapes. See link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types] on Red Hat Ecosystem Catalog portal.
+
+.Procedure
+
+. Log in to your link:https://cloud.oracle.com/a/[Oracle Cloud Infrastructure] account that has administrator privileges, and create a child compartment. Note the name and the link:https://docs.oracle.com/en-us/iaas/Content/libraries/glossary/ocid.htm[{ocid-first}] of the compartment. See link:https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/process-automation/oci-process-automation/create-compartment.html[Create a compartment] in the Oracle documentation.
```

Comment:
I'm not going to do a full peer review. Just a suggestion that you rewrite that first sentence to remove "that has ..." and remove the unnecessary comma. "Note" > "Record".